onnxscript/ir/_core.py

```diff
@@ -1999,6 +2003,12 @@ def __iter__(self) -> Iterator[Node]:
     def __reversed__(self) -> Iterator[Node]:
         return reversed(self._nodes)
 
+    def _set_input_and_initializer_value_names_into_name_authority(self):
```

Comment:
Feel free to add a todo for me to trigger this when inputs and initializers are modified

Reply:
Added